.github/workflows/ci.yml

```diff
@@ -20,14 +20,17 @@ jobs:
     name: "${{ matrix.compiler }} (Boost: ${{ matrix.boost }})"
     steps:
       - uses: actions/checkout@master
+      - name: update package list
+        run: apt-get update
       - name: Install dependencies
         run: apt-get install -y libcurl4-openssl-dev
       - name: Install Boost
         if: ${{ matrix.boost == true }}
-        run: apt-get install -y libboost-system1.65-dev libboost-test1.65-dev libboost-program-options1.65-dev
+        run: apt-get install -y libboost-system1.71-dev libboost-test1.71-dev libboost-program-options1.71-dev
       - name: Build
         run: script/ci_build.sh
-
+      - name: Install
+        run: cd build && make install
```

Comment:
Maybe it's a good idea to use the same call as below.

Reply:
done!